ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/views/PaywallView.kt

```diff
@@ -97,12 +104,22 @@ class PaywallView : FrameLayout {
         addView(
             ComposeView(context).apply {
                 setContent {
-                    val paywallOptions = PaywallOptions.Builder { dismissHandler?.invoke() }
-                        .setListener(internalListener)
-                        .setFontProvider(fontProvider)
-                        .setOfferingId(offeringId)
-                        .setShouldDisplayDismissButton(shouldDisplayDismissButton ?: false)
-                        .build()
+                    var paywallOptions by remember {
+                        mutableStateOf(PaywallOptions.Builder { dismissHandler?.invoke() }
+                            .setListener(internalListener)
+                            .setFontProvider(fontProvider)
+                            .setOfferingId(offeringId)
+                            .setShouldDisplayDismissButton(shouldDisplayDismissButton ?: false)
+                            .build())
+                    }
+                    updateOfferingId = {
```

Comment:
nit: Sorry to bother you guys again, but I highly recommend staying away from unstable lambdas, as it may cause unforeseen recompositions and performance hits.

Reply:
Hi @Jjastiny, I believe in this case it should be fine since we are only storing a lambda to trigger the update from outside the compose context, but I agree it's probably not the cleanest approach. I just pushed another approach inspired by your approach that also seems to work fine. cc @vegaro